libdnf/hy-package.cpp

```diff
@@ -1053,6 +1053,7 @@ dnf_package_get_advisories(DnfPackage *pkg, int cmp_type)
     dataiterator_init(&di, pool, 0, 0, UPDATE_COLLECTION_NAME,
                       pool_id2str(pool, s->name), SEARCH_STRING);
     dataiterator_prepend_keyname(&di, UPDATE_COLLECTION);
+    dataiterator_prepend_keyname(&di, UPDATE_COLLECTIONLIST);
```

Comment:
Why it requires to set `UPDATE_COLLECTION` and `UPDATE_COLLECTIONLIST`?

Reply:
I believe it because we are searching for string `s->name` by iterating over `UPDATE_COLLECTION_NAME` and the solv structure is:
 - `UPDATE_COLLECTIONLIST`:
 - - `UPDATE_COLLECTION`:
 - - - `UPDATE_COLLECTION_NAME`: some-name